Sources/Client/BaseURL.swift

```diff
@@ -12,21 +12,24 @@ import Foundation
 public struct BaseURL: CustomStringConvertible {
     static let placeholderURL = URL(string: "https://getstream.io")!
     
+    public static let usEast = BaseURL(serverLocation: .usEast)
+    public static let dublin = BaseURL(serverLocation: .dublin)
```

Comment:
```suggestion
    public static let usEast = BaseURL(url: URL(string: "https://chat-proxy-us-east.stream-io-api.com/")!)
    public static let dublin = BaseURL(url: URL(string: "https://chat-proxy-dublin.stream-io-api.com/")!)
```
and remove `ServerLocation`